sentry/src/main/java/io/sentry/ScopeType.java

```diff
@@ -4,7 +4,5 @@ public enum ScopeType {
   CURRENT,
   ISOLATION,
   GLOBAL,
-
-  // TODO [HSM] do we need a combined as well so configureScope
```

Comment:
Yes, at least for cross platform SDKs until they support different scope types separately themselves.